pkg/remote/trans/default_server_handler.go

```diff
@@ -34,12 +34,15 @@ import (
 
 // NewDefaultSvrTransHandler to provide default impl of svrTransHandler, it can be reused in netpoll, shm-ipc, framework-sdk extensions
 func NewDefaultSvrTransHandler(opt *remote.ServerOption, ext Extension) (remote.ServerTransHandler, error) {
-	svcInfo := GetDefaultSvcInfo(opt.SvcMap)
+	if opt.WithOnlyAcceptingHTTP2Traffic {
```

Comment:
What scenario do we need this option

Reply:
If there is a conflicting method name between services, users need to specify the fallback service for the method, otherwise, an error will occur when server initialization.
But when using http2 traffic, it doesn't matter if there are any conflicting method names between services because http2 logic does not use method svc map.
If the server side users enable this option, if there are any conflicting method names, no error will occur, but server-side users should note that only http2 traffic will be accepted instead.